package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@beefree.io/sdk",
-  "version": "4.1.0",
+  "version": "4.2.0-feat-media-library.2",
```

Comment:
won't it be a braking change? just asking didn't really reason about it

Reply:
it introduces just a new method and a new interface for the file manager